tests/headless/test_cdn.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import requests
+
+from kuma.core.urlresolvers import reverse
```

Comment:
Functional tests should not import from Kuma, or run under a Django environment. These URLs need to be hard-coded.